src/docs/guides/typed-components.md

```diff
@@ -5,20 +5,27 @@ url: /docs/typed-components
 contributors:
   - adamdbradley
   - manucorporat
+  - rwaskiewicz
 ---
 
 # Typed Components
 
-Web Components generated with stencil come with typing information automatically generated by the Stencil compiler. Typescript declarations provide strong guarantees when consuming components:
+Web Components generated with Stencil come with typing information automatically generated by the Stencil compiler.
+
+In general, Typescript declarations provide strong guarantees when consuming components:
 
 - Ensuring that proper values are passed down as properties
-- Code autocompletion in modern IDEs such as Atom and VSCode
+- Code autocompletion in modern IDEs such as VSCode
 - Events' details
 - Signature of components' methods
 
-This public types are automatically generated by Stencil in `src/component.d.ts`, which allows for strong typing in JSX (just like React) and `HTMLElement` interfaces for each component.
+These public types are automatically generated by Stencil in `src/component.d.ts`.
+This file allows for strong typing in JSX (just like React) and `HTMLElement` interfaces for each component.
+It is recommended that this file be checked in with the rest of your code in source control .
```

Comment:
should this line maybe we an 'info' or 'note' call-out box, just to make it a little obvious?

Reply:
👍 done in 211bf458d

Reply:
![Screen Shot 2022-08-22 at 4 00 06 PM](https://user-images.githubusercontent.com/1930213/186008225-650b601e-4a3b-4723-92cb-fcbb425648fa.png)

Reply:
beautiful 💯